### PR TITLE
feat: include chart name and version in commits for pages chart repos

### LINKED ISSUE #879: Chart pages should include a more useful commit message

### PR DESCRIPTION
**Pull request by haysclark**
Resolves #879

Signed-off-by: Hays Clark <hays.clark@gmail.com>

### CODE REVIEW

**Comment by haysclark** [comment]
Removed some dead code which was also causing Duplicate code warning in SonarCloud.

File: pkg/cmd/helm/release/testdata/pages/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Chart Repository
```
Comment (ankitm123) [inline pkg/cmd/helm/release/testdata/pages/README.md:1]: Not sure if this should be in testdata folder.

**Pull request merged**